amplify/backend/function/stockerlambda/src/app.js

```diff
@@ -64,4 +64,30 @@ app.get('/api/prices/live', async (req, res) => {
   res.json(result);
 });
 
+app.get('/api/price', async (req, res) => {
```

Comment:
This new endpoint allows us to retrieve historical prices for a given ticker. Will be useful for transaction form exchange rates too